lab-jaren/test/game-router-test.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Game = require('../model/game.js');
+const storage = require('../lib/storage.js');
+const superagent = require('superagent');
+
+const apiURL = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+describe('testing /api/games', function() {
+  describe('testing POST', function() {
+    describe('with valid input', function() {
```

Comment:
This'll be a style preference, but I find it's a lot easier to put explicit status numbers in my describe/it blocks so i can find that easier when I run my tests, i.e. I'd put something like "should return a game with status 200", etc.